packages/webchat/src/components/messages/renderer/Carousel.tsx

```diff
@@ -2,8 +2,9 @@ import React from 'react'
 import Slider from 'react-slick'
 
 // Added those manually to remove the font dependencies which keeps showing 404 not found
-import '../../../../../../assets/slick/slick-theme.css'
-import '../../../../../../assets/slick/slick.css'
+// TODO: see what to do with these
+// import '../../../../../../assets/slick/slick-theme.css'
+// import '../../../../../../assets/slick/slick.css'
```

Comment:
This will go on a CDN. Keep all the files from /assets in a folder in the repository, we'll need them when running the webchat locally and we'll just push the whole assets folder on the CDN